modules/ding_base/ding_base.default_environment_indicator_environments.inc

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * @file
+ * ding_base.default_environment_indicator_environments.inc
+ */
+
+/**
+ * Implements hook_default_environment_indicator_environment().
+ */
+function ding_base_default_environment_indicator_environment() {
+  $export = array();
+
+  $environment = new stdClass();
+  $environment->disabled = FALSE; /* Edit this to true to make a default environment disabled initially */
+  $environment->api_version = 1;
+  $environment->machine = 'prod';
+  $environment->name = 'Production';
+  $environment->regexurl = '.*';
+  $environment->settings = array(
+    'color' => '#000000',
+    'text_color' => '#ffffff',
+    'weight' => '3',
+    'position' => 'top',
+    'fixed' => 0,
+  );
+  $export['prod'] = $environment;
+
+  $environment = new stdClass();
+  $environment->disabled = FALSE; /* Edit this to true to make a default environment disabled initially */
+  $environment->api_version = 1;
+  $environment->machine = 'stg';
+  $environment->name = 'Staging';
+  $environment->regexurl = '.*.stg.easysuite.dk';
+  $environment->settings = array(
+    'color' => '#c48000',
+    'text_color' => '#ffffff',
+    'weight' => '2',
+    'position' => 'top',
+    'fixed' => 0,
+  );
+  $export['stg'] = $environment;
+
+  $environment = new stdClass();
+  $environment->disabled = FALSE; /* Edit this to true to make a default environment disabled initially */
+  $environment->api_version = 1;
+  $environment->machine = 'dev';
+  $environment->name = 'Development';
+  $environment->regexurl = '.*.dev.easysuite.dk';
```

Comment:
is this environment related to some other dev instead the one used by the MD team. I mean we use _*.easysuite.dev.inlead.dk_ and not _'.*.dev.easysuite.dk_ or this url is for some other dev environment?

Reply:
In the future they will have dev sites for their own use, hosted on this domain.